prefect_dbt/cli/configs/snowflake.py

```diff
@@ -17,13 +21,20 @@ class SnowflakeTargetConfigs(TargetConfigs):
 
     Args:
         credentials: The credentials to use to authenticate; if there are
-            overlapping keys between credentials and TargetConfigs,
-            e.g. schema, credentials takes precedence.
+            duplicate keys between credentials and TargetConfigs,
+            e.g. schema, an error will be raised.
     """
 
+    type: str = "snowflake"
+    schema_: str = Field(default=None, alias="schema")
```

Comment:
Nice to have the type default to snowflake, and schema_ gets yielded from credentials